Source/WebCore/Modules/mediasource/MediaSourceInterfaceWorker.cpp

```diff
@@ -98,7 +98,7 @@ bool MediaSourceInterfaceWorker::attachToElement(WeakPtr<HTMLMediaElement>&& ele
     m_handle->setHasEverBeenAssignedAsSrcObject();
     m_handle->ensureOnDispatcher([element = WTFMove(element)](MediaSource& mediaSource) mutable {
         mediaSource.attachToElement(WTFMove(element));
-    });
+    }, true);
```

Comment:
true is a bit mysterious, how about adding `bool forceRun = true` to make it clear?